### PR TITLE
insurance contract

### PR DESCRIPTION
**Pull request by rantifamilusi**
# Description

This PR includes the following proposed change(s):

- https://justice.gov.bc.ca/jira/browse/FAMS3-2030

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/InsuranceClaim.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        public string ClaimType { get; set; }
+
+        [Description("Claim Status")]
+        public string ClaimStatus { get; set; }
+
+        [Description("claim number")]
+        public string ClaimNumber { get; set; }
+
+        [Description("claim center")]
+        public string ClaimCenter { get; set; }
```
Comment (peggy-gov-bc-ca) [inline app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/InsuranceClaim.cs:19]: claimCenter is a string?

File: app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/InsuranceClaim.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        public Name Adjustor { get; set; }
+
+        [Description("Adjustor Phone")]
+        public Phone AdjustorPhone { get; set; }
+
+        [Description("Identifer involved in insurance claim")]
+        public IEnumerable<PersonalIdentifier> Identifiers { get; set; }
+
+        [Description("Person involved")]
+        public Name InsuredPerson { get; set; }
```
Comment (peggy-gov-bc-ca) [inline app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/InsuranceClaim.cs:31]: InsuredParty is better , and it should be a collection.

Comment (rantifamilusi) [reply app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/InsuranceClaim.cs:31]: I think insuredPeople is better cos it properly identify what the object it as against using the word Party? Yes it should be a collection.

File: app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/InsuranceClaim.cs (hunk 1 above)
Comment (rantifamilusi) [reply app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/InsuranceClaim.cs:19]: We need to remove this and wait for feedback from Locate Services in the weekly checkin

**Comment by rantifamilusi** [comment]
> some comments in contract.
> Besides that,
> 
> 1. Update Person contract
> 2. Update Sample project

What does 1 mean ? the contract is already updated

**Pull request merged**